### ISSUE TITLE
[Bug]: Img2Img Generate button stops working in Inpaint mode

### ISSUE THREAD

**Issue by rytt0001** [anapnoe/stable-diffusion-webui-ux#108]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What happened?

the generate button of the img2img tab stop responding when clicked and doesn't launch a new generation.
It does It in Inpaint mode but not in base Img2Img mode.

Webui is running on linux Manjaro, while accessed on Windows 10.

### Steps to reproduce the problem

1. Go to Img2Img tab then select Inpaint
2. send an image to the inpaint mode and then draw a mask

3. to produce the bug you need to  write some text in the prompt box and immediately click on the generate button, while the token counter is refreshing

then if done right, the generate will become the interrupt/skip buttons for a few seconds before returning to generate, without launching a new generation

### What should have happened?

launched a new generation

### Commit where the problem happens

079a5e4d8c8a3a4de43f75a7f64a8f363d02a8c1

### What platforms do you use to access the UI ?

Windows

### What browsers do you use to access the UI ?

Brave

### Command Line Arguments

```Shell
--listen --show-negative-prompt --disable-safe-unpickle --deepdanbooru --api --skip-install --always-batch-cond-uncond --no-half-vae --medvram --opt-sdp-no-mem-attention
```

### List of extensions

https://github.com/DominikDoom/a1111-sd-webui-tagcomplete
https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111.git
https://github.com/ljleb/prompt-fusion-extension.git
https://github.com/thomasasfk/sd-webui-aspect-ratio-helper.git
https://github.com/Mikubill/sd-webui-controlnet
https://github.com/hnmr293/sd-webui-cutoff
https://github.com/hako-mikan/sd-webui-regional-prompter.git
https://github.com/yfszzx/stable-diffusion-webui-images-browser
https://github.com/arenatemp/stable-diffusion-webui-model-toolkit
https://github.com/ilian6806/stable-diffusion-webui-state.git
https://github.com/Coyote-A/ultimate-upscale-for-automatic1111.git

### Console logs

```Shell
Nothing in the console and nothing in the browser console,
```

**Comment by anapnoe** [comment]
gradio edit image component is inefficient and buggy it uses 5 canvases instead of 1 the input resolution is 2160 x 3840 is huge sending 5 times the data to the backend for processing then sending them back to the inpaint view gradio is planning to rewrite the image edit component put there is no milestone for this I am thinking to bypass it and copy the buffers directly without the need to roundtrip the backend

**Comment by ghostsquad** [comment]
Ya, I'm seeing the same thing, and it essentially causes my browser to stop responding entirely.

**Comment by ghostsquad** [comment]
Why does this work fine though in upstream, but not in this fork?

**Comment by ankithjoseph** [comment]
after recent commit same issue with text2img , the generate button stops responding when clicked and doesn't launch a new generation. first few generation works fine,( guess it has something to do with queue. )

**Comment by ghostsquad** [comment]
I've had to move back to automatic1111, but I'm on an older commit (`22bcc7be1` Mar 29) for stability reasons.

**Comment by anapnoe** [comment]
the info provided is so little to reproduce the issue maybe it is a racing problem you can disable the livepreview from settings or use Aprox NN in preview mode 
@ghostsquad the commit that works for you has a different gradio version

**Comment by ghostsquad** [comment]
@anapnoe correct. I have both automatic1111 installed at `22bcc7be1` as well as this fork installed.

**Comment by ghostsquad** [comment]
any update on this?
